frontend/src/main/scala/bloop/data/ClientInfo.scala

```diff
@@ -292,7 +292,7 @@ object ClientInfo {
                   val dirName = clientDir.underlying.getFileName().toString
                   val attrs =
                     Files.readAttributes(clientDir.underlying, classOf[BasicFileAttributes])
-                  val isOldDir = attrs.creationTime.toInstant.isBefore(deletionThresholdInstant)
+                  val isOldDir = attrs.lastModifiedTime.toInstant.isBefore(deletionThresholdInstant)
```

Comment:
I think it's better to use last modified and it's possible to check in the tests actually.